cmd/distro.go

```diff
@@ -678,3 +685,72 @@ func NewDistroReportCmd() (*cobra.Command, error) {
 	distroReportCmd.Flags().String("name", "", "the distro name")
 	return distroReportCmd, nil
 }
+
+func NewDistroExportCmd() (*cobra.Command, error) {
```

Comment:
This function will never return an error, it will always return nil as the error. Please either handle the error or change the function signature to reflect that it won't fail.

Same thing applies to the other functions